### ISSUE TITLE
Only working in Chrome

### ISSUE THREAD

**Issue by clabough** [samjulien/elements-demo#1]
If I download this, install packages, build, and then use static-server, it works in Chrome, but not Firefox, Edge, or IE on Windows. I even tried hard-coding to all of your package versions and still the same.

I noticed you had two of these in angular.json, but removing one didn't help.

`"input": "node_modules/document-register-element/build/document-register-element.js"`

**Comment by samjulien** [comment]
Custom elements are currently only fully supported by Chrome. You can enable them with a flag in Firefox, but they are not at all supported by IE or Edge. You can see [here](https://caniuse.com/#search=custom%20elements%20v1) for more info on browser support of custom elements. Cheers!

**Comment by clabough** [comment]
Yes, I understood that, but I thought that was the point of the polyfill.

**Comment by clabough** [comment]
Got it working in other browsers by removing `encapsulation: ViewEncapsulation.Native`
